hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ledger/BalanceChange.java

```diff
@@ -115,18 +117,24 @@ public static BalanceChange changingNftOwnership(
     }
 
     public static BalanceChange tokenAdjust(final Id account, final Id token, final long amount) {
```

Comment:
IMO these `hbarAdjust` and `tokenAdjust` factories should be renamed to like `hbarCustomFeeAdjust()` to reduce the risk someone calls them outside a custom fee context.

Reply:
I was thinking the same thing 🤔 ✅

Reply:
DONE